pre-docs/bolt_configure_orchestrator.md

```diff
@@ -41,11 +41,15 @@ Warning: By granting users access to Bolt tasks, you give them permission to
 run arbitrary commands and upload files as a super-user.  In the PE console,
 click Access control > User roles.
 
+#. Make sure the bolt task is installed in the `production` environment on the
+   master.
 #. From the list of user roles, click the one you want to have Bolt task
    permissions.
 #. On the Permissions tab, in the Type box, select Tasks.
-#. For Permission, select Run tasks, and then select bolt from the Object list.
+#. For Permission, select Run tasks, and then select `bolt` from the Object list.
 #. Click Add permission, and then commit the change.
+#. You may uninstall the `bolt` module from the production environment if you
```

Comment:
That's awkward.

Reply:
I'm having trouble coming up with clearer wording. I can remove this and step one if we just say the bolt module must be installed in the production environment?

Reply:
No, the description is fine. It's just awkward that you have to add and remove the module in production if you want to use it in other environments.

Reply:
Lets discuss what to do with BOLT-440. I think maybe it's worth just telling people they need to install the bolt module into production always.